src/ossl/ecc.rs

```diff
@@ -709,3 +714,153 @@ impl Verify for EccOperation {
         Ok(self.output_len)
     }
 }
+
+fn ckd_to_digest_type_name(mech: CK_EC_KDF_TYPE) -> *const c_char {
```

Comment:
nit: this should probably be ckd_to_digest_name, without "type"

Reply:
Thanks. Fixed.